.github/workflows/pre-commit.yml

```
@@ -0,0 +1,16 @@
+name: Run linters
+
+on:
+  push:
+    branches: [ main ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)